### ISSUE TITLE
Implicitly changing the target file document when saving with a different filename

### ISSUE THREAD

**Issue by rstm-sf** [EvotecIT/OfficeIMO#49]
Implicitly changing the loaded file document when saving with a different filename. I think changing the target document should not change the current `WordDocument` object. This behavior was introduced due to a document cloning issue https://github.com/OfficeDev/Open-XML-SDK/issues/1139

We can see this

https://github.com/EvotecIT/OfficeIMO/blob/e965a042489a79aa04f630aaf6bb186a3b8284c8/OfficeIMO.Word/WordDocument.cs#L481

https://github.com/EvotecIT/OfficeIMO/blob/e965a042489a79aa04f630aaf6bb186a3b8284c8/OfficeIMO.Word/WordDocument.cs#L588-L599

**Comment by PrzemyslawKlys** [comment]
@rstm-sf Could we implement a workaround that if "Load" is used a copy to temp first is made so that we don't destroy the source until Open-XML-SDK gets a fix? Not sure if the same applies to loading from a stream.

**Comment by PrzemyslawKlys** [comment]
Right, but since most of the packing is part of Open-XML-SDK it should be fixed there I guess... and in the meantime we could do some workaround so that OfficeIMO behaves as expected. I was thinking on simply - if we are loading file from path, copy it somewhere else, do the magic and depending if user does Save or SaveAs save it into expected place. Not great, but like you said - proper implementation would take time and is probably outside of OfficeIMO (as far as I understand. Unless you propose to do refactoring here - but then my brain will most likely explode and I will not be able to deliver :-p Someone else will have to do the heavy lifting.

**Comment by PrzemyslawKlys** [comment]
Should be resolved by:
- https://github.com/EvotecIT/OfficeIMO/pull/452
